stories/status-indicator/status-indicator.stories.js

```diff
@@ -396,7 +396,7 @@ export const StatusIndicatorLabels = () => `
 			</path>
 		</svg>	
 	</svg>
-	<span class="fd-status-indicator--lg__text fd-status-indicator--positive__text">100%</span>
+	<span class="fd-status-indicator__label--lg">100%</span>
```

Comment:
Should be:
```
<span class="fd-status-indicator__label fd-status-indicator__label--lg">100%</span>
```

we already discussed that a modifier class like `fd-status-indicator__label--lg` should not be used alone but with a base element class like `fd-status-indicator__label`. In this case `fd-status-indicator__label--lg`  is modifying the props set in `fd-status-indicator__label`.